src/cpp/desktop/DesktopMainWindow.cpp

```diff
@@ -105,6 +105,11 @@ MainWindow::MainWindow(QUrl url) :
    connect(webView(), SIGNAL(onCloseWindowShortcut()),
            this, SLOT(onCloseWindowShortcut()));
 
+   connect(&desktopInfo(), &DesktopInfo::fixedWidthFontListChanged, [this]() {
+      QString js = QStringLiteral("if (window.onFontListReady) window.onFontListReady()");
```

Comment:
Nit: I usually do these with e.g. `typeof window.onFontListReady === "function"` so there's no risk of trying to execute a non-function (probably moot in this case since we control both sides!)

Reply:
I think that's worth doing as well, even more since it makes the intent even more clear.